diff --git a/libgcc/Makefile.in b/libgcc/Makefile.in
index e86e98d..ada9242 100644
--- a/libgcc/Makefile.in
+++ b/libgcc/Makefile.in
@@ -41,7 +41,7 @@ long_double_type_size = @long_double_type_size@
 decimal_float = @decimal_float@
 enable_vtable_verify = @enable_vtable_verify@
 enable_decimal_float = @enable_decimal_float@
-fixed_point = @fixed_point@
+#fixed_point = @fixed_point@
 
 host_noncanonical = @host_noncanonical@
 target_noncanonical = @target_noncanonical@
