diff --git a/libgcc/Makefile.in b/libgcc/Makefile.in
index ed51a2cb5..b0cc36d49 100644
--- a/libgcc/Makefile.in
+++ b/libgcc/Makefile.in
@@ -42,7 +42,7 @@ long_double_type_size = @long_double_type_size@
 decimal_float = @decimal_float@
 enable_vtable_verify = @enable_vtable_verify@
 enable_decimal_float = @enable_decimal_float@
-fixed_point = @fixed_point@
+#fixed_point = @fixed_point@
 with_aix_soname = @with_aix_soname@
 solaris_ld_v2_maps = @solaris_ld_v2_maps@
 enable_execute_stack = @enable_execute_stack@
